packages/core/src/lib/MockLog.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Log} from '..';
+
+/**
+ * An utility class for testing used for get the logging messages without printing to the user.
+ */
+export class MockLog implements Log {
+    /**
+     * Creates a new MockLog instance
+     * @param tag the tag used when logging. Printed at the beggining of a log message.
+     * @param verbose if the Log is verbose. Debug messages are only printed on verbose logs.
+     */
+    public verbose: boolean;
```

Comment:
It doesn't seem like `verbose` and `tag` are used in the implementation. Should they exist here?

Reply:
I thought about removing them, but when I read about promps I saw that every prompt class should have them.
I wasn't sure what the best practice is but what I thought here is: if every prompt needs to have it, the best practice is to put it in the interface so if we want to build another prompt class for some reason we will have a proper interface for it

Reply:
and if we will remove them, some functions that will use `prompt` could give it `tag` or `verbose`, and if we will not have it in our implementation we couldn't give it `MockPrompt` instead